File: app/src/main/java/com/bq/zowi/controllers/SessionController.java
package com.bq.zowi.controllers;

import org.jetbrains.annotations.Nullable;

/* JADX INFO: loaded from: classes.dex */
public interface SessionController {
    @Nullable
    String loadActiveZowiDeviceAddress();

    String loadActiveZowiName();

    String loadDefaultZowiName();

    boolean hasDismissedWizard();

    void resetActiveZowi();

    void saveActiveZowiDeviceAddress(String str);

    void saveActiveZowiName(String str);

    void saveWizardDismissed(boolean dismissed);
}

File: zowi-core/src/main/kotlin/com/bq/zowi/api/SessionController.kt
package com.bq.zowi.api

import io.reactivex.Single

interface SessionController {
    fun loadActiveZowiDeviceAddress(): String?
    fun loadActiveZowiName(): String
    fun loadDefaultZowiName(): String
    fun hasDismissedWizard(): Boolean
    fun resetActiveZowi()
    fun saveActiveZowiDeviceAddress(address: String)
    fun saveActiveZowiName(name: String)
    fun saveWizardDismissed(dismissed: Boolean)
}
